src/devices/device_models/AC_branches.jl

```diff
@@ -120,14 +108,69 @@ function branch_rate_constraint(ps_m::CanonicalModel,
 
     rating_constraint(ps_m,
                         range_data,
-                        Symbol("rate_limit_fwd_$(B)"),
-                        (Symbol("Pbr_fwd_$(B)"), Symbol("Qbr_fwd_$(B)")))
+                        Symbol("RateLimitFT_$(B)"),
+                        (Symbol("FpFT_$(B)"), Symbol("FqFT_$(B)")))
 
     rating_constraint(ps_m,
                         range_data,
-                        Symbol("rate_limit_bwd_$(B)"),
-                        (Symbol("Pbr_bwd_$(B)"), Symbol("Qbr_bwd_$(B)")))
+                        Symbol("RateLimitTF_$(B)"),
+                        (Symbol("FpTF_$(B)"), Symbol("FqTF_$(B)")))
 
     return
 
 end
+
+function branch_rate_constraint(ps_m::CanonicalModel,
+    devices::PSY.FlattenIteratorWrapper{B},
+    device_formulation::Union{Type{StaticLineUnbounded}, Type{StaticTransformerUnbounded}},
+    system_formulation::Type{S}) where {B <: PSY.ACBranch,
+                                        S <: PM.AbstractPowerFormulation}
+
+    # This code is intended to do nothing
+
+    return
+
+end
+
+
+function branch_flow_constraint(ps_m::CanonicalModel,
+                                devices::PSY.FlattenIteratorWrapper{PSY.MonitoredLine}, # TODO: Do we need an AbstractMonitoredLine?
```

Comment:
No we don't really need more abstraction here.

Reply:
I thought that the entire point was to dispatch on the abstract type above the concrete types so that we support user extension. Without that abstraction, users always have to use the struct `MonitoredLine` to invoke this constraint.